tests/README.md

```diff
@@ -24,7 +24,8 @@ docker compose -f e2e-tests-docker-compose.yml run e2e-tests
 ```
 ## go to root dir of the project, e2e-tests-aarch64.Dockerfile is located there
 docker build -t spin-e2e-tests -f e2e-tests-aarch64.Dockerfile .
-MYSQL_IMAGE=arm64v8/mysql:8.0.32 REDIS_IMAGE=arm64v8/redis:6.0-alpine3.17 docker compose         \
+MYSQL_IMAGE=arm64v8/mysql:8.0.32 REDIS_IMAGE=arm64v8/redis:6.0-alpine3.17 POSTGRES_IMAGE=arm64v8/postgres:14.7 \
+    docker compose         \
```

Comment:
(Comment on file itself)

Should other areas of the README be updated to instruct on the delineation of running the `fermyon-platform` tests vs the `spinup-tests`, etc?  Also the doc appears to still reference [make test-integration](https://github.com/fermyon/spin/blob/main/tests/README.md?plain=1#L5) which seems outdated?

Reply:
The e2e tests are still evolving and there are some integration tests that I am going to pull into e2e tests. I am going to update this document once the scope of integration tests is more concise.

Reply:
Sounds good, thanks @rajatjindal